resources.whatwg.org/build/deploy.sh

```diff
@@ -89,6 +89,7 @@ MATCHES=$(
   grep -niE '\s+$' "$INPUT_FILE" | perl -lpe 'print "\nTrailing whitespace:" if $. == 1'
   grep $'\t' "$INPUT_FILE" | perl -lpe 'print "\nTab:" if $. == 1'
   grep $'\xc2\xa0' "$INPUT_FILE" | perl -lpe 'print "\nUnescaped nonbreaking space:" if $. == 1'
+  grep $'[≪≫]' fetch.bs | perl -lpe 'print "\nWrong list literals:" if $. == 1'
```

Comment:
I think it would be okay to run this for all standards. Should we also list the ones to use instead in the error message? I guess one problem might be that not all terminals are UTF-8 yet...

Reply:
Changed to use escaped unicode